dwave/cloud/client.py

```diff
@@ -221,10 +224,17 @@ def from_config(cls, config_file=None, profile=None, client=None,
             token (str, default=None):
                 API authorization token.
 
-            solver (str, default=None):
-                Default :term:`solver` to use in :meth:`~dwave.cloud.client.Client.get_solver`.
-                If undefined, :meth:`~dwave.cloud.client.Client.get_solver` will return the
-                first solver available.
+            solver (dict/str, default=None):
+                Default :term:`solver` features to use in :meth:`~dwave.cloud.client.Client.get_solver`.
+
+                Defined via dictionary of solver feature constraints
+                (see :meth:`~dwave.cloud.client.Client.get_solvers`). For backward
```

Comment:
For backward compatibility, a solver name, as a string, is also accepted and converted to ``{"name": <solver name>}``.